docs/spreadsheet/how-to-retrieve-a-list-of-the-hidden-worksheets-in-a-spreadsheet.md

```diff
@@ -18,112 +18,22 @@ ms.localizationpriority: medium
-    public static List<Sheet> GetHiddenSheets(string fileName)
-```
-
-### [Visual Basic](#tab/vb-0)
-```vb
-    Public Function GetHiddenSheets(ByVal fileName As String) As List(Of Sheet)
-```
-***
-
-
-The method works with the workbook you specify, filling a **[List\<T\>](https://msdn2.microsoft.com/library/6sh2ey19)** instance with a reference to each hidden **Sheet** object.
-
-## Calling the GetHiddenSheets method
-
-The method returns a generic list that contains information about the individual hidden **Sheet** objects. To call the **GetHiddenWorksheets** method, pass the required parameter value, as shown in the following code.
-
-### [C#](#tab/cs-1)
-```csharp
-    // Revise this path to the location of a file that contains hidden worksheets.
-    const string DEMOPATH = 
-        @"C:\Users\Public\Documents\HiddenSheets.xlsx";
-    List<Sheet> sheets = GetHiddenSheets(DEMOPATH);
-    foreach (var sheet in sheets)
-    {
-        Console.WriteLine(sheet.Name);
-    }
-```
-
-### [Visual Basic](#tab/vb-1)
-```vb
-    ' Revise this path to the location of a file that contains hidden worksheets.
-    Const DEMOPATH As String =
-        "C:\Users\Public\Documents\HiddenSheets.xlsx"
-    Dim sheets As List(Of Sheet) = GetHiddenSheets(DEMOPATH)
-    For Each sheet In sheets
-        Console.WriteLine(sheet.Name)
-    Next
-```
-***
-
-
-## How the code works
-
-The following code starts by creating a generic list that will contain information about the hidden worksheets.
-
-### [C#](#tab/cs-2)
-```csharp
-    List<Sheet> returnVal = new List<Sheet>();
-```
-
-### [Visual Basic](#tab/vb-2)
-```vb
-    Dim returnVal As New List(Of Sheet)
-```
-***
-
-
-Next, the following code opens the specified workbook by using the **SpreadsheetDocument.Open** method and indicating that the document should be open for read-only access (the final **false** parameter value). Given the open workbook, the code uses the **WorkbookPart** property to navigate to the main workbook part, storing the reference in a variable named **wbPart**.
-
-### [C#](#tab/cs-3)
-```csharp
-    using (SpreadsheetDocument document = 
-        SpreadsheetDocument.Open(fileName, false))
-    {
-        WorkbookPart wbPart = document.WorkbookPart;
-        // Code removed here… 
-    }
-    return returnVal;
-```
-
-### [Visual Basic](#tab/vb-3)
-```vb
-    Using document As SpreadsheetDocument =     SpreadsheetDocument.Open(fileName, False)
-        Dim wbPart As WorkbookPart = document.WorkbookPart
-        ' Code removed here…
-    End Using
-    Return returnVal
-```
-***
-
+You can use the **GetHiddenSheets** method, to retrieve a list of the hidden worksheets in a workbook. The **GetHiddenSheets** method accepts a single parameter, a string that indicates the path of the file that you want to examine. The method works with the workbook you specify, filling a **[List\<T\>](https://msdn2.microsoft.com/library/6sh2ey19)** instance with a reference to each hidden **Sheet** object.
```

Comment:
```suggestion
You can use the **GetHiddenSheets** method, to retrieve a list of the hidden worksheets in a workbook. The **GetHiddenSheets** method accepts a single parameter, a string that indicates the path of the file that you want to examine. The method works with the workbook you specify, filling a **[List\<T\>](/dotnet/api/system.collections.generic.list-1)** instance with a reference to each hidden **Sheet** object.
```

Reply:
We can do a pass to convert all these to more modern links (and then to xrefs if we can get that enabled) later